src/common/tokens/operations.js

```diff
@@ -11,13 +11,26 @@ const loadTokens = () => async (dispatch, getState) => {
 
 const addToken = contractAddress => async dispatch => {
 	const tokenService = getGlobalContext().tokenService;
-	const tokenInfo = await tokenService.getTokenInfo(contractAddress);
-	await tokenService.addToken(tokenInfo);
+	try {
+		const tokenInfo = await tokenService.getTokenInfo(contractAddress);
+		await tokenService.addToken(tokenInfo);
+	} catch (error) {
+		await dispatch(
```

Comment:
@eduardosmoura could you add logging like I did in this PR https://github.com/SelfKeyFoundation/Identity-Wallet/pull/967?

Reply:
Yes sure @rodrigopavezi